docs/tutorials/tabular_prediction/tabular-faq.md

```diff
@@ -0,0 +1,22 @@
+# FAQ
+:label:`sec_faq`
+
+How can I perform inference on a file that won't fit in memory?
+
+The Tabular Dataset API works with pandas Dataframes, which supports chunking data in to sizes that fit in memory.
+Here's an example of one such chunk-based inference:
+
+```{.python .input}
+train_data = task.Dataset(file_path='https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv')
+predictor = task.fit(train_data=train_data)
+
+r = requests.get('https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv', allow_redirects=True)
+open('test.csv', 'wb').write(r.content)
+reader = pd.read_csv('test.csv', chunksize=32)
```

Comment:
I'd recommend a larger chunk size for the example, such as 1000. 32 will be slow as currently the models are loaded from disk for every call to predict to ensure memory safety.